platforms/extensibility/test-kit/build.gradle.kts

```diff
@@ -53,14 +53,13 @@ dependencies {
 // TODO Find a way to not register this and the task instead
 configurations.remove(configurations.apiStubElements.get())
 
+val runtimeApiInfoDir = layout.buildDirectory.dir("generated-resources/runtime-api-info")
 val generateTestKitPackageList by tasks.registering(PackageListGenerator::class) {
-    classpath.from(sourceSets.main.map { it.runtimeClasspath })
-    outputFile = layout.buildDirectory.file("runtime-api-info/test-kit-relocated.txt")
+    classpath.from(configurations.runtimeClasspath.get().elements)
```

Comment:
`.elements` should not be necessary here. A configuration is already a FileCollection